src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Program.MinimalAPIs.OrgOrIndividualB2CAuth.cs

```diff
@@ -107,10 +107,12 @@
 
     var user = await graphServiceClient.Me.Request().GetAsync();
 
+    var today = DateOnly.FromDateTime(DateTime.Now);
+
     var forecast =  Enumerable.Range(1, 5).Select(index =>
```

Comment:
Nit; for consistency with the change above:

```suggestion
    var today = DateOnly.FromDateTime(DateTime.Now);
    var forecast =  Enumerable.Range(1, 5).Select(index =>
```